### PR TITLE
save invalid xml into db

### PR DESCRIPTION
**Pull request by xlqian**
save invalid xml into db
http://jira.canaltp.fr/browse/NAVITIAII-1975

### CODE REVIEW

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by xlqian** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
Is it possible to add a test on that?

**Comment by kinnou02** [comment]
hey @xlqian, don't forget this!

**Comment by xlqian** [comment]
@pbougue difficult to do that, the test will involve a db...

**Comment by pbougue** [comment]
Not sure about the difficulty, you can create a messed up xml and check that it is ok with a query of sqlalchemy (see https://github.com/CanalTP/kirin/blob/master/tests/integration/ire_test.py#L223)

**Comment by xlqian** [comment]
Ok, I'm going with that! Thx

**Comment by xlqian** [comment]
Jenkins, test this please

File: tests/integration/ire_test.py [hunk 1]
```diff
@@ -369,3 +369,18 @@ def test_ire_trip_removal_parity(mock_rabbitmq):
...
+    """
+    bad_ire = get_ire_data('bad_ire.xml')
+    res = api_post('/ire', data=bad_ire, check=False)
+    assert res[1] == 400
+    assert res[0]['message'] == 'Invalid arguments'
+    with app.app_context():
+        assert len(RealTimeUpdate.query.all()) == 1
+        assert RealTimeUpdate.query.first().status == 'KO'
+        assert RealTimeUpdate.query.first().error == \
+            'invalid xml, impossible to find "Train" in xml elt InfoRetard'
```
Comment (pbougue) [inline tests/integration/ire_test.py]: Is it possible add a test on the raw_data? The most important is that we store the input... :sweat:

**Comment by pbougue** [comment]
Ok for me, thanks!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Pull request merged**